src/App.js

```diff
@@ -1,15 +1,22 @@
 import React from 'react';
+import Article from './components/Article';
+
+const App = () => (
+  <div>
```

Comment:
if you are using a wrapper for more than 1 component, it is better to use `<fragment></fragment>` or its' shortened version - `<></>`